aemanalyser-core/src/main/java/com/adobe/aem/analyser/AemPackageConverter.java

```diff
@@ -41,6 +41,17 @@
 
 public class AemPackageConverter {
 
+    private static final Map<String, String> DEFAULT_NAMESPACE_MAPPINGS = Map.of(
```

Comment:
No need to define JCR defaults nor sling (https://github.com/apache/sling-org-apache-sling-feature-cpconverter/blob/e861e1e6838acbc9128349a510e4ead26804b150/src/main/java/org/apache/sling/feature/cpconverter/vltpkg/JcrNamespaceRegistry.java#L52)

Reply:
Updated in [a50d6d1](https://github.com/adobe/aemanalyser-maven-plugin/pull/205/commits/a50d6d1c88a4f8d5a8436106dd251488b2294bc7), thanks!